### PR TITLE
adding logline to slide component

### PR DESCRIPTION
**Pull request by jeremie74**

### CODE REVIEW

**Review by GrandSchtroumpf — CHANGES_REQUESTED** [review]

File: libs/movie/src/lib/movie/components/slide/slide.component.html [hunk 1]
```diff
@@ -2,8 +2,9 @@ <h4 *ngIf="movieView.directors">{{ movieView.directors | displayName }}</h4>
 <h1 *ngIf="movieView.titles.international" class="mat-display-2">{{ movieView.titles.international }}</h1>
 <h2 *ngIf="movieView.titles.original">{{ movieView.titles.original }}</h2>
 <movie-title-features [movie]="title"></movie-title-features>
-<ng-content select="movie-slide-cta"></ng-content>
-<p *ngIf="movieView.synopsis" class="mat-body-2">
-    {{ movieView.synopsis | maxLength: 332}}
+<span *ngIf="!movieView.logline" class="fixBanner"></span>
+<p *ngIf="movieView.logline" class="mat-body-2">
+    {{ movieView.logline | maxLength: 332}}
```
[the remark this answers is not in the document]
Comment (jeremie74) [reply libs/movie/src/lib/movie/components/slide/slide.component.html:7]: It's not 33 but 332 i don't understand.

[the remark this answers is not in the document]
Comment (GrandSchtroumpf) [reply libs/movie/src/lib/movie/components/slide/slide.component.html:7]: Sorry typo: "I don't *think* logline should be limited to *332* characters".
You can remove the maxLength pipe altogether. And if it's not used in the rest of the component, remove the `MaxLengthModule` too.

File: libs/movie/src/lib/movie/components/slide/slide.module.ts [hunk 2]
```diff
@@ -6,18 +6,15 @@ import { FlexLayoutModule } from '@angular/flex-layout';
...
 
 @NgModule({
     imports: [
         CommonModule,
         FlexLayoutModule,
-
         // Blockframes
         MovieTitleFeaturesModule,
-        DisplayNameModule,
         MaxLengthModule
```
Comment (GrandSchtroumpf) [inline libs/movie/src/lib/movie/components/slide/slide.module.ts:18]: You're not using it in the template anymore. Please remove it

Comment (jeremie74) [reply libs/movie/src/lib/movie/components/slide/slide.module.ts]: Its done maybee you hadn't the last commit

**Review by GrandSchtroumpf — APPROVED** [review]

**Comment by jeremie74** [comment]
Is there a responsive version or we shoud

**Pull request merged**